test/JDBC/expected/catalogs_dbo_sys_schema-upgrade-vu-verify.out

```diff
@@ -0,0 +1,6872 @@
+varchar
+login_myschematest_master
+login_myschematest_mydb
+login_myschematest_tempdb
+~~END~~
+
+select name from master..syslogins where name like 'login_myschematest_%' order by 1
+go
+~~START~~
+varchar
+login_myschematest_master
+login_myschematest_mydb
+login_myschematest_tempdb
+~~END~~
+
+select name from .master..syslogins where name like 'login_myschematest_%' order by 1
+go
+~~START~~
+varchar
+login_myschematest_master
+login_myschematest_mydb
+login_myschematest_tempdb
+~~END~~
+
+select name from tempdb.dbo.syslogins where name like 'login_myschematest_%' order by 1
+go
+~~START~~
+varchar
+login_myschematest_master
+login_myschematest_mydb
+login_myschematest_tempdb
+~~END~~
+
+select name from .tempdb.dbo.syslogins where name like 'login_myschematest_%' order by 1
+go
+~~START~~
+varchar
+login_myschematest_master
+login_myschematest_mydb
+login_myschematest_tempdb
+~~END~~
+
+select name from tempdb.sys.syslogins where name like 'login_myschematest_%' order by 1
+go
+~~START~~
+varchar
+login_myschematest_master
+login_myschematest_mydb
+login_myschematest_tempdb
+~~END~~
+
+select name from .tempdb.sys.syslogins where name like 'login_myschematest_%' order by 1
+go
+~~START~~
+varchar
+login_myschematest_master
+login_myschematest_mydb
+login_myschematest_tempdb
+~~END~~
+
+select name from tempdb..syslogins where name like 'login_myschematest_%' order by 1
+go
+~~START~~
+varchar
+login_myschematest_master
+login_myschematest_mydb
+login_myschematest_tempdb
+~~END~~
+
+select name from .tempdb..syslogins where name like 'login_myschematest_%' order by 1
+go
+~~START~~
+varchar
+login_myschematest_master
+login_myschematest_mydb
+login_myschematest_tempdb
+~~END~~
+
+select name from myschematest_mydb.dbo.syslogins where name like 'login_myschematest_%' order by 1
+go
+~~START~~
+varchar
+login_myschematest_master
+login_myschematest_mydb
+login_myschematest_tempdb
+~~END~~
+
+select name from .myschematest_mydb.dbo.syslogins where name like 'login_myschematest_%' order by 1
+go
+~~START~~
+varchar
+login_myschematest_master
+login_myschematest_mydb
+login_myschematest_tempdb
+~~END~~
+
+select name from myschematest_mydb.sys.syslogins where name like 'login_myschematest_%' order by 1
+go
+~~START~~
+varchar
+login_myschematest_master
+login_myschematest_mydb
+login_myschematest_tempdb
+~~END~~
+
+select name from .myschematest_mydb.sys.syslogins where name like 'login_myschematest_%' order by 1
+go
+~~START~~
+varchar
+login_myschematest_master
+login_myschematest_mydb
+login_myschematest_tempdb
+~~END~~
+
+select name from myschematest_mydb..syslogins where name like 'login_myschematest_%' order by 1
+go
+~~START~~
+varchar
+login_myschematest_master
+login_myschematest_mydb
+login_myschematest_tempdb
+~~END~~
+
+select name from .myschematest_mydb..syslogins where name like 'login_myschematest_%' order by 1
+go
+~~START~~
+varchar
+login_myschematest_master
+login_myschematest_mydb
+login_myschematest_tempdb
+~~END~~
+
```

Comment:
sysprocesses test cases are missing.

Reply:
sysprocesses was removed because the contents are not predictable: sometimes process keep hanging around during the JDBC tests.

Reply:
added test cases similar to sysforeignkeys